src/site/xdoc/index.xml

```diff
@@ -86,6 +86,13 @@
 
 		</section>
 
+		<section name="HAPI HL7v2 2.5.2 Released - May 15, 2024">

```

Comment:
I think you'll want to add a changelog as well. See this similar MR (https://github.com/hapifhir/hapi-hl7v2/commit/193a2ec7051439654feaadaef5b327aec3eb604b#diff-480762ed7e0cb6175cb14bd289b7167eeac7ae35cf253725155eed73398da621)

Reply:
Oh THAAATS where the change log is. I was looking everywhere for it!